src/pages/experiments/[experimentId]/data-exploration/components/differential-expression-tool/DiffExprCompute.jsx

```diff
@@ -95,20 +99,40 @@ const DiffExprCompute = (props) => {
 
   // Validate form when the groups selected changes.
   useEffect(() => {
+
+    // Check if `compareWith` is in the same group as `cellSet`
```

Comment:
why do we need to do this?

Reply:
Per Vicky's comment on the needs to keep comparison in-group, not required anymore, so removed in latest commit.